docs/operate-a-stake-pool/hardware-requirements.md

```diff
@@ -9,11 +9,11 @@ The latest technical specifications and supported platforms can be found on the
 
 Currently the following specifications are recommended:
 
-- Servers: 1 for block producer node, at least 1 for relay node
+- Servers: 1 for block producer node, at least 1 as relay node
```

Comment:
I thought today about bumping it to 2 instead, and then also about all the pools that are in violation of it, as well as who benefits from the "recommendations" (not the new SPO on a limited budget).  For now I've changed it to 2 in the interest of pushing this through (@os11k please respond if you have any objection).

My heart tells me this number should be a lowest common denominator, which tends to provide flexibility.  If I had to choose between one relay with 12 cores and 2 relays with the "recommended" 2 cores 😨 I would rather have the single relay.  But yes it would be more in keeping with the other changes to use 2 consistently recommended by IOG elsewhere.

Reply:
>  If I had to choose between one relay with 12 cores and 2 relays with the "recommended" 2 cores

Well worse could get to 1 relay with 2 cores (the idea with 2 is primarily redundancy), also wouldn't want folks could use dev portal reference as "CF recommends 1 relay" 😄  - but ye, as you said it is primarily to keep recommendations consistent

Reply:
sure, let's not tell them you can just add the BP keys to your 1 relay and _really_ slash your pool running costs 😆